snap/component.go

```diff
@@ -29,11 +29,12 @@ import (
 
 // ComponentInfo contains information about a snap component.
 type ComponentInfo struct {
-	Component   naming.ComponentRef `yaml:"component"`
-	Type        ComponentType       `yaml:"type"`
-	Version     string              `yaml:"version"`
-	Summary     string              `yaml:"summary"`
-	Description string              `yaml:"description"`
+	Component           naming.ComponentRef `yaml:"component"`
+	Type                ComponentType       `yaml:"type"`
+	Version             string              `yaml:"version"`
+	Summary             string              `yaml:"summary"`
+	Description         string              `yaml:"description"`
+	ComponentProvenance string              `yaml:"provenance,omitempty"`
```

Comment:
why not just Provenance?

Reply:
So we can have a `Provenance` method, we do something similar with `snap.Info.SnapProvenance`.